opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -1346,6 +1346,15 @@ namespace Opm {
 
         // We use the rates from the privious time-step to reduce oscilations
         WellGroupHelpers::updateWellRates(fieldGroup, schedule(), reportStepIdx, previous_well_state_, well_state_);
+        {
+            // HACK: Set ALQ for off-process wells to zero
+            const auto& all_wells = schedule().getWells(reportStepIdx);
+            for (const auto& w : all_wells) {
```

Comment:
Here you can just use the well names:
```
for (const auto& wname : schedule().wellNames(reportStepIdx)) {
   ...
}
```

Reply:
Amended and pushed.

Reply:
Just a note: I had to go back to getting the wells, since I needed to call `well.isProducer()`.